src/prefs.c

```diff
@@ -604,6 +604,9 @@ static void prefs_init_dialog(void)
 	widget = ui_lookup_widget(ui_widgets.prefs_dialog, "check_scroll_stop_at_last_line");
 	gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(widget), editor_prefs.scroll_stop_at_last_line);
 
+	widget = ui_lookup_widget(ui_widgets.prefs_dialog, "check_highlight_current_word");
+	gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(widget), editor_prefs.highlight_current_word);
+
```

Comment:
Ok, I'll try it. But if this code isn't needed and can be done by stash, why is it all here?